.github/workflows/ci.yaml

```diff
@@ -214,7 +214,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        java: [ 11, 17, 21 ]
+        java: [ 11, 17, 21, 23 ]
```

Comment:
Is the plan to just keep this as a matrix of Java LTS versions + latest Java release?

Reply:
Yes, I think so. To be more precise, the latest supported JDK in Graal SDK.